pages/common/git-merge-repo.md

```diff
@@ -0,0 +1,13 @@
+# git merge-repo
+
+> Merge two repository histories.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-merge-repo>.
+
+- Merge repository's branch into current repository's directory:
+
+`git merge-repo {{local-path-to-repo.git}} {{branch-name}} {{path/to/directory}}`
```

Comment:
```suggestion
`git merge-repo {{local_path_to_repo.git}} {{branch_name}} {{path/to/directory}}`
```
Mandatory change: snake case is [required](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).

```suggestion
`git merge-repo {{local/path/to/repo.git}} {{branch_name}} {{path/to/directory}}`
```

Optional change: no such style guide requirement exists.